ORStools/proc/isochrones_layer_proc.py

```diff
@@ -82,9 +86,22 @@ def __init__(self):
                 name=self.IN_RANGES,
                 description="Comma-separated ranges [min or m]",
                 defaultValue="5, 10"
-            )
+            ),
+            QgsProcessingParameterNumber(
+                name=self.IN_SMOOTHING,
+                description="Smoothing factor between 0 [fine grained] and 100 [concave hull]",
+                defaultValue=None,
+                minValue=0,
+                maxValue=100,
+                optional=True
+            ),
+
         ]
 
+        # Set flag of smoothness parameters to advanced
+        smooth_param = next((i for i in self.PARAMETERS if i.name() == self.IN_SMOOTHING), None)
+        smooth_param.setFlags(smooth_param.flags() | QgsProcessingParameterDefinition.FlagAdvanced)
```

Comment:
All other advanced Params are set in https://github.com/GIScience/orstools-qgis-plugin/blob/4d9e4ad0e19a93637ac748174d0c20f47a578ea5/ORStools/proc/base_processing_algorithm.py#L209
with the info on what params are advanced defined in 
https://github.com/GIScience/orstools-qgis-plugin/blob/4d9e4ad0e19a93637ac748174d0c20f47a578ea5/ORStools/common/__init__.py#L52

If there's no good reason to deviate from this, I'd suggest moving this there :)